src/main/resources/config/liquibase/changelog/20220401174637_added_entity_Gene.xml

```diff
@@ -20,8 +20,33 @@
             <column name="hugo_symbol" type="varchar(255)">
                 <constraints nullable="true" />
             </column>
+            <column name="hgnc_id" type="varchar(255)">
+                <constraints nullable="true" />
+            </column>
             <!-- jhipster-needle-liquibase-add-column - JHipster will add columns here -->
         </createTable>
+
+        <!-- This data was generated on 4/4/2022 12:00PM EST - Please update this when csv file is modified -->
+        <loadData
+            file="config/liquibase/data/gene.txt"
+            separator="|"
+            tableName="gene"
+            usePreparedStatements="true">
+        </loadData>
```

Comment:
I'm still including the data in this PR, will create another PR to clean up the data usage per today's discussion.